src/interfaces/extrinsic/signedExtensions/index.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019-2020 Centrality Investments Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { ExtDef, ExtInfo, ExtTypes } from './types';
+
+import cennznetExtensions from './cennznet';
```

Comment:
wondering why/if we need these signedExtension files, can't we just patch `ChargeTransactionPayment` and it will be ok?

Reply:
I tried with just patching ChargeTransactionPayment, instead.. but it didn't work..

Reply:
Maybe something hardcoded in polkadot for these calls
```
      ...registry.getSignedExtensionTypes(),
      ...registry.getSignedExtensionExtra(),
```